Description: Initial
 Some non-sense shit.
 .
 debian-patch-best-practice (1.0.0) jammy; urgency=medium
 .
   * Initial release.
Author: Atlas Yu <atlas.yu@canonical.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2023-03-23

--- /dev/null
+++ b/makefile
@@ -0,0 +1,13 @@
+RLIB=target/release/libdebian_patch_best_practice.rlib
+
+all: ${RLIB}
+
+${RLIB}:
+	cargo build --release
+
+install: ${RLIB}
+	install -d /usr/lib/
+	install ${RLIB} /usr/lib/libdpbp_add.rlib
+
+clean:
+	rm -rf target Cargo.lock
